package.json

```diff
@@ -43,6 +52,8 @@
     "tslint": "^5.1.0"
   },
   "dependencies": {
+    "@types/node-emoji": "^1.4.0",
```

Comment:
You only need this as a dependency if something you export relies on a type from it.